jme3-examples/src/main/java/jme3test/asset/TestIssue2083.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Please include the standard JME license header.

Reply:
Assuming this is newly-written code, "2023" will do fine for the copyright date.